src/routes/index.js

```diff
@@ -1,8 +1,8 @@
 import express from 'express';
-import apiRouter from './api';
+import apiRouter from './api/passwordResetRoute';
 
 const index = express.Router();
 
-index.use('/api', apiRouter);
+index.use('/api/v1/users', apiRouter);
```

Comment:
I believe other routes for the API are set in the `api/index.js`. You can set your specific route there instead of updating the existing one here.